app/Http/Controllers/RedirectController.php

```diff
@@ -12,8 +12,27 @@
 class RedirectController extends Controller
 {
     private array $allowedDomains = [
-        'news.retroachievements.org',
+        // First-party
+        'api-docs.retroachievements.org',
         'docs.retroachievements.org',
+        'media.retroachievements.org',
+        'news.retroachievements.org',
+        'retroachievements.org',
+
+        // Third-party
+        'backloggd.com',
+        'backloggery.com',
+        'completionist.me',
+        'exophase.com',
+        'github.com',
+        'howlongtobeat.com',
+        'infinitebacklog.net',
+        'infinitebacklog.nl',
+        'psnprofiles.com',
+        'steamcommunity.com',
+        'twitch.tv',
+        'twitter.com',
+        'youtube.com',
```

Comment:
What about `www.google.com`? Official forum posts link to google searches for GameFAQs, longplays, and wikipedia

Reply:
Possibly `discord.com`? I see several posts linking to stuff on discord - particularly for events.

Reply:
Do we need explicit subdomains? I see both "https://youtube.com" and "https://www.youtube.com" links.

Reply:
I don't think Google should be on the allow list. The "I'm Feeling Lucky" button and google.com/url can redirect users to arbitrary websites.

Reply:
I've added `discord.com` to the allowlist. No strong feelings one way or the other towards `google.com`. 

@Jamiras

> Do we need explicit subdomains?

Fortunately, `www` as an explicit subdomain is not required. On this branch, try navigating to the following URLs and you should see the same result:

http://localhost:64000/redirect?url=https://github.com
http://localhost:64000/redirect?url=https://www.github.com

Reply:
I understand that github.com will resolve to www.github.com. My question is whether we need to put both github.com and www.github.com in the list so both URLs will be trusted and avoid the redirect warning.

Reply:
Unless I'm misinterpreting the question, having only `github.com` in the list results in both `github.com` and `www.github.com` being trusted.

This is demonstrable by having it in the list and navigating to either of the two routes given above. A redirect warning will never be shown; the user will be sent directly to GitHub.

Reply:
Nope. That was my question. I just saw all the subdomains for retroachievements.org and assumed www would have to be explicitly called out.

Reply:
Not sure if I added those subdomain variants for brevity or ignoring my own code which explicitly omits subdomains 😅 
https://github.com/RetroAchievements/RAWeb/blob/master/app/Http/Controllers/RedirectController.php#L37

Reply:
Resolved merge conflict & removed all first-party subdomains. I've confirmed those work fine without needing to be included.